nilearn/datasets/struct.py

```diff
@@ -497,6 +510,11 @@ def fetch_icbm152_brain_gm_mask(data_dir=None, threshold=0.2, resume=True,
                                 n_iter=2, verbose=1):
     """Downloads ICBM152 template first, then loads the 'gm' mask.
 
+    The default template of :term:`fMRIPrep` is the asymmetrical ICBM152 2009,
```

Comment:
If you have the exact same message in multiple docstrings, you can consider adding it to the standard documentation in `nilearn/_utils/docs.py`